include/wx/generic/grid.h

```diff
@@ -220,8 +220,8 @@ class WXDLLIMPEXP_CORE wxGridCellEditor : public wxGridCellWorker
     wxGridCellEditor();
 
     bool IsCreated() const { return m_control != NULL; }
-    wxControl* GetControl() const { return m_control; }
-    void SetControl(wxControl* control) { m_control = control; }
+    wxWindow* GetControl() const { return m_control; }
```

Comment:
Unfortunately this is not backwards compatible and will break the existing code doing
```cpp
wxControl* w = editor->GetControl();
```

We should try to avoid breaking compatibility unless absolutely unavoidable, so let's keep
```cpp
wxControl* GetControl() const { return wxDynamicCast(m_control, wxControl); }
```
and add
```cpp
wxWindow* GetWindow() const { return m_control; }
```

Of course, we also need to write comments explaining what's going on here and update the documentation in `interface/wx/grid.h`, mentioning that `GetControl()` is kept for compatibility only and `GetWindow()` should be used in the new code.